tests/test_pipreqs.py

```diff
@@ -37,50 +47,51 @@ def setUp(self):
             "bs4",
             "after_method_is_valid_even_if_not_pep8",
         ]
-        self.modules2 = ["beautifulsoup4"]
-        self.local = ["docopt", "requests", "nose", "pyflakes"]
-        self.project = os.path.join(os.path.dirname(__file__), "_data")
-        self.empty_filepath = os.path.join(self.project, "empty.txt")
-        self.imports_filepath = os.path.join(self.project, "imports.txt")
-        self.imports_no_version_filepath = os.path.join(self.project, "imports_no_version.txt")
-        self.imports_any_version_filepath = os.path.join(self.project, "imports_any_version.txt")
-        self.non_existent_filepath = os.path.join(self.project, "non_existent_file.txt")
-
-        self.parsed_packages = [
+        cls.modules2 = ["beautifulsoup4"]
+        cls.local = ["docopt", "requests", "nose", "pyflakes", "ipython"]
+        cls.project = os.path.join(os.path.dirname(__file__), "_data")
+        cls.empty_filepath = os.path.join(cls.project, "empty.txt")
+        cls.imports_filepath = os.path.join(cls.project, "imports.txt")
+        cls.imports_no_version_filepath = os.path.join(cls.project, "imports_no_version.txt")
+        cls.imports_any_version_filepath = os.path.join(cls.project, "imports_any_version.txt")
+        cls.non_existent_filepath = os.path.join(cls.project, "non_existent_file.txt")
+
+        cls.parsed_packages = [
             {"name": "pandas", "version": "2.0.0"},
             {"name": "numpy", "version": "1.2.3"},
             {"name": "torch", "version": "4.0.0"},
         ]
 
-        self.parsed_packages_no_version = [
+        cls.parsed_packages_no_version = [
             {"name": "pandas", "version": None},
             {"name": "tensorflow", "version": None},
             {"name": "torch", "version": None},
         ]
 
-        self.parsed_packages_any_version = [
+        cls.parsed_packages_any_version = [
             {"name": "numpy", "version": None},
             {"name": "pandas", "version": "2.0.0"},
             {"name": "tensorflow", "version": None},
             {"name": "torch", "version": "4.0.0"},
         ]
 
-        self.project_clean = os.path.join(os.path.dirname(__file__), "_data_clean")
-        self.project_invalid = os.path.join(os.path.dirname(__file__), "_invalid_data")
-        self.parsed_packages = [
-            {"name": "pandas", "version": "2.0.0"},
-            {"name": "numpy", "version": "1.2.3"},
-            {"name": "torch", "version": "4.0.0"},
-        ]
-        self.empty_filepath = os.path.join(self.project, "empty.txt")
```

Comment:
was this line deleted in this commit? if so, why? is it related to warning suppression?